widget/view.html

```diff
@@ -271,34 +271,38 @@ <h3 class="modal-title col-md-9">FortiAI Configuration</h3>
                   </select>
                 </div>
               </div>
-              <div class="row">
-                <div class="col-md-6 margin-top-md">
-                  <label for="projectId" class="control-label configuration-selection"> Project Id <span
-                      class="required">*</span>
-                  </label>
-                  <input id="projectId" type="text" class="form-control" name="projectId" data-ng-model="projectId"
-                    placeholder="projectId">
-                </div>
-              </div>
               <div class="row">
                 <div class="col-md-6 margin-top-md">
                   <label for="playbookAssistantName" class="control-label configuration-selection"> Playbook Assistant
                     Name</label>
                   <input id="playbookAssistantName" type="text" class="form-control" name="playbookAssistantName"
-                    data-ng-model="playbookAssistantName">
+                    data-ng-model="playbookAssistantName" placeholder="fsr-ai-playbook-assitant">
                 </div>
               </div>
               <div class="row">
                 <div class="col-md-6 margin-top-md">
                   <label for="socAssistantName" class="control-label configuration-selection"> SOC Assistant
                     Name</label>
                   <input id="socAssistantName" type="text" class="form-control" name="socAssistantName"
-                    data-ng-model="socAssistantName">
+                    data-ng-model="socAssistantName" placeholder="fsr-ai-soc-assitant">
                 </div>
               </div>
               <div class="col-md-12 margin-top-18 padding-0">
-                <button type="button" class="btn btn-primary" data-ng-click="createAssistant()"
-                  id="create-ai-assistants"><i class="icon icon-check margin-right-sm"></i>Create</button>
+                <button type="button" class="btn btn-primary float-left" data-ng-click="createUpdateAssistant()" data-ng-disabled="assistantCreated"
+                  id="create-ai-assistants"><i class="icon icon-check margin-right-sm"></i>{{assistantStatus}}</button>
+                  <!-- spinner -->
+                  <div class="margin-left-sm margin-right-sm install-progress-icon col-md-1" data-ng-show="createUpdateActionClicked">
+                    <div class="custom-installing-spinner-content">
+                        <div class="custom-installing-spinner-block">
+                            <div class="custom-installing-spinner">
+                                <i class="icon icon-installing-loader custom-installing-icon fa fa-spin green-check"></i>
+                            </div>
+                        </div>
+                    </div>
+                  </div>
+                  <label class="text-center font-size-12 green-check margin-left-md" data-ng-show="assistantCreated">
+                    Assistant {{assistantStatus}}d successfully.
```

Comment:
Assistant {{assistantStatus}}d successfully. --> Assistant {{assistantStatus}} successfully.

But what is this message is all about?